adoc/retail_chap_components.adoc

```diff
@@ -63,6 +63,13 @@ In most cases, different terminals will require a different operating system (OS
 For example, an information kiosk has a high-resolution touchscreen, where a cashier terminal might only have a very basic display.
 While both of these terminals require similar processing and network functionality, they will require different OS images to ensure that the different display mechanisms work correctly.
 
+1024 MB as minimum memory for hosts that need to run OS images built with Kiwi (PXE booted or not)
```

Comment:
The minimum memory requirement is 1024 MB for hosts that ...

Reply:
thanks!